src/main/scala/com/cognite/sdk/scala/v1/resources/functions.scala

```diff
@@ -0,0 +1,192 @@
+    requestSession.get[Items[Function], Items[Function]](
+      baseUrl,
+      value => value
+    )
+
+  override def createItems(items: Items[FunctionCreate]): F[Seq[Function]] = {
+    implicit val customPrinter: Printer = Printer.noSpaces.copy(dropNullValues = true)
+    requestSession.post[Seq[Function], Items[Function], Items[FunctionCreate]](
+      items,
+      baseUrl,
+      value => value.items
+    )
+  }
+
+  override def deleteByIds(ids: Seq[Long]): F[Unit] =
+    DeleteByIds.deleteByIds(requestSession, baseUrl, ids)
+
+  override def deleteByExternalIds(externalIds: Seq[String]): F[Unit] =
+    DeleteByExternalIds.deleteByExternalIds(requestSession, baseUrl, externalIds)
+
+  override def retrieveByIds(ids: Seq[Long]): F[Seq[Function]] =
+    RetrieveByIds.retrieveByIds(requestSession, baseUrl, ids)
+
+  override def retrieveByExternalIds(externalIds: Seq[String]): F[Seq[Function]] =
+    RetrieveByExternalIds.retrieveByExternalIds(requestSession, baseUrl, externalIds)
+}
+
+object Functions {
+  implicit val functionErrorDecoder: Decoder[FunctionError] = deriveDecoder[FunctionError]
+  implicit val functionDecoder: Decoder[Function] = deriveDecoder[Function]
+  implicit val functionsItemsDecoder: Decoder[Items[Function]] =
+    deriveDecoder[Items[Function]]
+  implicit val functionsItemsWithCursorDecoder: Decoder[ItemsWithCursor[Function]] =
+    deriveDecoder[ItemsWithCursor[Function]]
+  implicit val functionErrorEncoder: Encoder[FunctionError] = deriveEncoder[FunctionError]
+  implicit val createFunctionEncoder: Encoder[FunctionCreate] = deriveEncoder[FunctionCreate]
+  implicit val createFunctionsItemsEncoder: Encoder[Items[FunctionCreate]] =
+    deriveEncoder[Items[FunctionCreate]]
+
+  implicit val errorOrFunctionDecoder: Decoder[Either[CdpApiError, Items[Function]]] =
+    EitherDecoder.eitherDecoder[CdpApiError, Items[Function]]
+  implicit val errorOrUnitDecoder: Decoder[Either[CdpApiError, Unit]] =
+    EitherDecoder.eitherDecoder[CdpApiError, Unit]
+  implicit val deleteRequestWithRecursiveAndIgnoreUnknownIdsEncoder
+      : Encoder[ItemsWithRecursiveAndIgnoreUnknownIds] =
+    deriveEncoder[ItemsWithRecursiveAndIgnoreUnknownIds]
+}
+
+class FunctionCalls[F[_]](val requestSession: RequestSession[F], val functionId: Long)
+    extends WithRequestSession[F]
+    with BaseUrl {
+  import FunctionCalls._
+  override val baseUrl = uri"${requestSession.baseUrl}/functions/$functionId/calls"
+
+  def callFunction(data: Json): F[FunctionCall] =
+    requestSession.post[FunctionCall, FunctionCall, Json](
+      data,
+      uri"${baseUrl.toString().dropRight(1)}",
+      value => value
+    )
+
+  def read(): F[Items[FunctionCall]] =
+    requestSession.get[Items[FunctionCall], Items[FunctionCall]](
+      baseUrl,
+      value => value
+    )
+
+  def filter(filter: FunctionCallFilter): F[Items[FunctionCall]] = {
+    implicit val customPrinter: Printer = Printer.noSpaces.copy(dropNullValues = true)
```

Comment:
:/  They should allow nulls, as all the other parts of the API does. But since it's playground, I guess we'll have to let it slide.